backend/src/routes/api/storage-class/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { FastifyRequest } from 'fastify';
+import { K8sResourceCommon, KubeFastifyInstance, StorageClassConfig } from '../../../types';
+import { isHttpError } from '../../../utils';
+import { errorHandler } from '../../../utils';
+
+export async function updateStorageClassMetadata(
+  fastify: KubeFastifyInstance,
+  request: FastifyRequest<{ Body: StorageClassConfig }>,
+): Promise<{ success: boolean; error: string }> {
+  const params = request.params as { storageClassName: string };
```

Comment:
Can we not adjust the incoming request type so we don't have to cast here?

Reply:
sure, did that